File: tier1/csharp/2afe6b8b/Report.cs
using System.Collections.Generic;
using System.Diagnostics;

public static class Report
{
    public static string Build(string name)
    {
        var parameters = new Dictionary<string, string>();
        parameters["target"] = name;
        parameters["mode"] = "summary";
        return Run(parameters["target"]);
    }

    private static string Run(string target)
    {
        Process.Start("/bin/sh", "-c \"/usr/bin/report " + target + "\"");
        return target;
    }
}

File: tier1/java/dae62426/Report.java
import java.io.IOException;
import java.util.HashMap;
import java.util.Map;

public final class Report {
    public static String build(String name) {
        Map<String, String> params = new HashMap<>();
        params.put("target", name);
        params.put("mode", "summary");
        return run(params.get("target"));
    }

    private static String run(String target) {
        try {
            Runtime.getRuntime().exec("/usr/bin/report " + target);
        } catch (IOException e) {
            return "";
        }
        return target;
    }
}
